app/views/barbeque/job_definitions/new.html.haml

```diff
@@ -1,3 +1,9 @@
+- content_for(:title, 'New job definition - Barbeque')
+- content_for(:header) do
+  %ol.breadcrumb
+    %li= link_to('Home', root_path)
+    %li= link_to(@job_definition.app.name, app_path(@job_definition.app.id))
```

Comment:
`@job_definition.app` exists when user comes from "New definition" button in /apps/:app_id (I think this is normal route to the new definition page).
Added nil guard https://github.com/cookpad/barbeque/pull/48/commits/fb056cf9ad29d3b1817d88836b2dbc46c164bc71